pkg/ui/datafile.go

```diff
@@ -3526,7 +3526,7 @@ app.controller('ListPodsCtrl', [
 app.controller('ListReplicationControllersCtrl', [
   '$scope',
   '$routeParams',
-  'k8sApi',
+  'k8sv1Beta3Api',
```

Comment:
Before merging, you should be able to switch this to the v1 API which will be the default starting today.